managed/ui/src/components/certificates/Certificates.js

```diff
@@ -153,32 +208,26 @@ class Certificates extends Component {
         >
           <i className="fa fa-download"></i> Download Root CA Cert
         </MenuItem>
+        <MenuItem
+          onClick={() => { 
+            !deleteDisabled && this.deleteRootCertificate(payload?.uuid)
```

Comment:
I feel like `!deleteDisabled` check should happen in `deleteRootCertificate()`
and it's actually redundant as a button will be disabled anyways

Reply:
For some reason, the button click action handler was being called even when the button was 'Disabled', 
Done